twingly-url_cache.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "dalli", "~> 2.7"
   spec.add_dependency "retryable", "~> 2.0"
 
-  spec.add_development_dependency "bundler", "~> 1.10"
+  spec.add_development_dependency "bundler", ">= 1.10", "< 2.2"
```

Comment:
not sure if viable, but if we drop support for 2.5 too I don't think we need to bother about bundler here, as we can count on everyone having some version of bundler installed

Reply:
For fun I actually tried creating a new gem using `bundler gem <NAME> -t` and looked at the output. It seems like no requirements are added to the gemspec (not even bundler itself) so I wonder if this is some old copy pasta from some old gem best practices. :) The initial commit in this repo does not really tell.

Reply:
We try to drop support for Ruby 2.5 and and remove the bundler dependency then :)

Reply:
aaand it worked 👍